MAUI/Cartesian-Charts/StackedArea.md

```diff
@@ -66,27 +66,27 @@ StackingAreaSeries series1 = new  StackingAreaSeries()
 {
```

Comment:
@snehakPV , i asked to have single instence for view model, and use it. 

ViewModel viewModel = new ViewModel();


and then, it will be. 

ItemsSource = ViewModel.Data1

Reply:
Updated the changes as mentioned.